libs/pages/application/src/lib/feature/page-settings-domains-feature/crud-modal-feature/crud-modal-feature.tsx

```diff
@@ -25,6 +27,7 @@ export function CrudModalFeature(props: CrudModalFeatureProps) {
   })
   const dispatch = useDispatch<AppDispatch>()
   const loadingStatus = useSelector<RootState, LoadingStatus>((state) => getCustomDomainsState(state).loadingStatus)
+  const { setMustConfirmClickOutside } = useContext(ModalContext)
```

Comment:
You need to use like this `const { setMustConfirmClickOutside } = useModal()`

Reply:
Well, for this, I ask myself actually, it's a method from the context. useModal get this method from the context as well but it does not add anything else to it. So why not using directly the method from the context?
It's an open question, I don't find the answer so if you have it, I would love to hear it!